### PR TITLE
pkg/operator: Fix small linting errors and remove unneeded code.

### PR DESCRIPTION
**Pull request by timflannagan1**
This would remove an unneeded `else` condition when the accompanied `if` statement returns a value.

Removes any declared type when initializing a variable using the `var` keyword.

### CODE REVIEW

File: pkg/operator/http.go [hunk 1]
```diff
@@ -321,7 +321,7 @@ func writeResultsAsCSV(columns []api.ReportGenerationQueryColumn, results []pres
 func writeResultsResponseAsTabular(logger log.FieldLogger, name string, columns []api.ReportGenerationQueryColumn, results []presto.Row, w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "text/tab-separated-values")
 	w.Header().Set("Content-Disposition", fmt.Sprintf("attachment;filename=%s.tsv", name))
-	var padding int = 2
+	var padding = 2
```
Comment (chancez) [inline pkg/operator/http.go]: This could actually be `padding := 2`

**Comment by chancez** [comment]
Minor nit-pick since you're fixing these things anyways.

File: pkg/operator/http.go (hunk 1 above)
Comment (timflannagan1) [reply pkg/operator/http.go]: Fixed.

**Comment by timflannagan1** [comment]
I saw that in `pkg/operator/http.go` that we have `reportGenerationQuerieLister` field in the server structure and its references five times throughout that file. Is that a typo?

**Comment by chancez** [comment]
Should probably be Queries or Query

File: pkg/operator/http.go (hunk 1 above)
Comment (chancez) [reply pkg/operator/http.go]: I'm not seeing that?

Comment (timflannagan1) [reply pkg/operator/http.go]: hmm, that was strange. Updated, again.

**Comment by timflannagan1** [comment]
rebased the commits

**Pull request merged**